doc/pages/user/015-astarte_dashboard.md

```diff
@@ -0,0 +1,394 @@
+# Astarte Dashboard
+
+Astarte provides a built-in UI that can be used for managing Interfaces, Devices, and Triggers. The
+Dashboard simplifies the development phases of applications that make use of Astarte, as well as
+troubleshooting activities.
+
+You can browse the source code of the Dashboard software on its
+[GitHub repository](https://github.com/astarte-platform/astarte-dashboard).
+
+## Introduction
+
+The Astarte Dashboard is a Single Page Application to get an overview of and manage your Realm on
+any browser. The Dashboard is designed to be a quick and easy way to give you immediate feedback on
+your work and as a quick and intuitive way to configure your realm. It is not meant to be operated
+by end-users, rather by infrastructure maintainers, owners who need information on the system
+status, and those working on projects based on Astarte. It is shipped by default with the Astarte
+standard distribution.
+
+The Dashboard is a graphical frontend for Astarte APIs; `astartectl`,
```

Comment:
The Dashboard is a graphical frontend for Astarte APIs, much like the CLI tool `astartectl`

Reply:
I tried to rephrased, see if it's any better now.